helm-chart/values-dev.yaml

```diff
@@ -18,6 +18,9 @@ brokers:
 
 hermesFrontEndBaseUrl: https://hermes-dev.lco.global/
 
+hermesEmail: hermes@lco.global
```

Comment:
I know the email have to be secret, but for consistency's sake I have `HERMES_EMAIL_USERNAME` and `HEREMS_EMAIL_PASSWORD` defined as k8s secrets (in both dev and prod namespaces). I think keeping these values together and handling them consistently reduces cognitive complexity.